lib/src/vDraw_basic.cpp

```diff
@@ -241,40 +241,40 @@ void imuDraw::draw(cv::Mat &image, const ev::vQueue &eSet, int vTime)
         auto aep = is_event<IMUevent>(*qi);
 
         switch(aep->sensor) {
-        case 0: { //acc x
+        case IMUevent::ACC_Y: { //acc y
             auto p_end = centre - cv::Point(0, aep->value * lin_scaler);
             cv::line(image, centre, p_end, violet, 4);
             break; }
-        case 1: {//acc y
-            auto p_end = centre + cv::Point(aep->value * lin_scaler, 0);
+        case IMUevent::ACC_X: {//acc x
+            auto p_end = centre - cv::Point(aep->value * lin_scaler, 0);
             cv::line(image, centre, p_end, violet, 4);
             break;}
-        case 2: {//acc z
+        case IMUevent::ACC_Z: {//acc z
             auto p_end = centre + cv::Point(aep->value * lin_scaler * 0.71,
```

Comment:
Is this a z value being split between x and y in a 2d point structure? Why?

Reply:
just for visualisation - that makes the 3rd axis "into" the screen.